Core/Core/SwiftUIViews/CoreDatePicker.swift

```diff
@@ -0,0 +1,155 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct CoreDatePicker {
+
+    public static func pickDate(for date: Binding<Date?>, minDate: Date? = nil, maxDate: Date? = nil, from controller: UIViewController) {
```

Comment:
Maybe we could rename this function to better express what it will do e.g:`showDatePicker`

Reply:
changed it.